wiki/Tournaments/oPHF/2018/en.md

```diff
@@ -0,0 +1,278 @@
+| Streamers | ![][flag_PH] [aEverr](https://osu.ppy.sh/users/6027084), ![][flag_PH] [Ainyan](https://osu.ppy.sh/users/3770641), ![][flag_PH] [Deadpulse](https://osu.ppy.sh/users/8575527) |
+| Commentators | List to be finalized. |
+| Graphic Designer | ![][flag_PH] [Mira-san](https://osu.ppy.sh/users/4469895) |
+| Statisticians | ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895), ![][flag_PH] [MapleSyrup-](https://osu.ppy.sh/users/1192936) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/801475)
+- [Livestream](https://www.twitch.tv/osuliveph)
+- [osu!PH Facebook Group](https://web.facebook.com/groups/OSU.Philippines/)
+
+------------------------
+
+## Participants
+
+*Qualified participants will be added in this section.*
+
+------------------------
+
+## Mappools
+
+*Mappools will be revealed one week in advance of each stage.*
+
+------------------------
+
+## Match Results
+
+*All match results will be added in this section.*
+
+------------------------
+
+## Rules
+
+### Game Modes and Scoring Format
+
+- The tournament will host the following game modes: osu!standard, osu!taiko, osu!catch, osu!mania 4K, and osu!mania 7K. In addition, the tournament will use **Score v2** as its scoring format. This requires players to play on the following osu! Builds: Stable (Latest), Beta, and Cutting Edge.
+
+### Tournament Formats
+
+- The **osu!standard** division will adopt a Qualifier Round (explained below) in order to reduce the field into 32 players which will then be subjected to a **single elimination** bracket. Seeding for this division will be based on qualifier results.
+
+- The **osu!taiko** division will be subjected to a **single elimination** bracket. Seeding for this division will be based on Performance Points as of 2018-11-03 (18:00 UTC+8:00).
+
+- The **osu!catch** division will adopt a Qualifier Round (explained below) and will be subjected to a **16-player double elimination** bracket. Seeding for this division will be based on qualifier results.
+
+- Both **osu!mania 4K** and **osu!mania 7K** divisions will adopt a Qualifier Round (explained below) in order to reduce the field to 16 players. Players who advance will be subjected to a **double elimination** bracket. Seeding for this division will be based on qualifier results.
+
+### Mods Used in the Tournament
+
+#### osu!standard
+
+- Rules for mod usage are indicated below. The mod will be applied if the beatmap chosen is included in a specific category in the mappool.
+

```

Comment:
Delete spacing.
Spacing only apply to separate headers. Apply to others if there's any.